trite-osprey/database/server.js

```diff
@@ -0,0 +1,64 @@
+const express = require('Express')
+const app = express();
+const pgp = require('pg-promise')()
+const bodyParser = require('body-parser')
+const{Beermo} = require('../backend/db.js')
+const cors = require('cors')
+// Parses the text as JSON and exposes the resulting object on
+app.use(bodyParser.json())
+//this turns of the encoding of the response body
+app.use(bodyParser.urlencoded({extended:false}))
+//the glue between express and react server
+app.use(cors())
+
+app.get('/beers', function(req,res) {
+  Beermo.getAll()
+  .then((results) =>
+    res.json(results)
+  )
+})
+
+app.get('/beers/:id', function(req,res) {
+  const id = req.params.id
+  Beermo.getOne(id)
+  .then((results) =>
+    res.json(results)
+  )
+})
+
+app.delete('/:id', function(req,res) {
+  const id = req.params.id
+  Beermo.deleteOne(id)
+  .then(() =>
+    res.json({1:'success'})
```

Comment:
`{1:'success'}` is an odd things to render. Consider rendering the deleted record or nothing with the correct status code.